repos/c2sm/packages/py-gt4py/package.py

```diff
@@ -49,6 +50,7 @@ class PyGt4py(PythonPackage):
     depends_on('py-numpy@1.24.2: ~blas ~lapack', type=('build', 'run'))
     depends_on('py-packaging@20.0:', type=('build', 'run'))
     depends_on('py-pybind11@2.5:', type=('build', 'run'))
```

Comment:
Can't you remove py-bind11?
We do not support cartesian anyway.

Reply:
okay.. good point. let me double check with the gt4py folks.